aviatrix/resource_aviatrix_saml_endpoint.go

```diff
@@ -151,6 +155,8 @@ func resourceAviatrixSamlEndpointRead(d *schema.ResourceData, meta interface{})
 				log.Printf("[WARN] Error setting 'rbac_groups' for (%s): %s", d.Id(), err)
 			}
 		}
+	} else {
+		d.Set("access_set_by", []string{})
```

Comment:
"access_set_by" is TypeString but here you are trying to set it to slice of string. Is that intended?

Reply:
Good catch.. It should be rbac_groups. corrected it